### DISCUSSION TITLE
Sztornó bizonylatok vs GT

### DISCUSSION THREAD

**Discussion by penztargepnagyker** [nav-gov-hu/eRECEIPT#26] [Ötletek]
Sok felhasználó fordul pénztárgép szervizekhez, hogy egy kiállított nyugta amikor sztornózásra kerül nem vonódik le a GT-ből. Erre valami megoldás? Esetleg létre lehetne hozni egy nGT mezőt, ahol a sztornózások összegét összesítenénk? Továbbá nem csak GT-nél van ez a gond, hanem a napi záráson az ÁFA bontásnál sem vonjuk le jelenleg a forgalomból a sztornózott/visszáruzott bizonylatok esetén.

Továbbá sérelmezik, hogy  meg kell adni minden esetben a vásárló adatokat, holott pl egy "téves bevitel" esetén nem feltétlen van vásárló, lehet, hogy csak "a macska átszaladt a pénztárgépen" illetve a vásárló nem feltétlen adja meg a nevét, címét a pénzárosnak.

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
